data/changelog.txt

```diff
@@ -1,33 +1,461 @@
+resolve issues with subdermal carbon filament
+Display IME candidate list and composition text correctly on Windows
+Fix dark gray in the ncurses client for terminal emulators that support 256 colors.
+Damage done on collisions now scales based on seatbelt quality
+Small Riot armor description fix
+fixes some size weight and value discrepancies with all the acorns
+Corrected S&W 619 description
+Fixed a typo in intercom mission dialogue
+Misc. Umbrella fixes
+Stop rigid boat hull breaking into pliable fabric sheets
+Adjust trapdoor spider dens
+Fixes mobile home park road connections
+Set tileset overmap to default true
+Changed shark_suit and shark_suit_faraday materials to plastic for the head/feet and lc_steel_chain in other slots. Also updated their weights to 18lbs.
+Now you can (once again?) wear rings under rigid gloves
+Reduce amount of platinum to craft platinum grille
+Flush map buffers after failing to create starting location
+Make meteorologist less likely to die
+Trapdoor spider den mapgen clearing flags
+Fixes Trenchcoat inconsistency and adds coverage to upper legs
+Change metal tanks crafting category
+Fix litcig burn out gated behind one-in-10 random chance
+Smart controller one engine support
+Calcium carbide production overhaul
+Weird interaction between skintight clothing and thick natural armor
+Fix paint requirements for some recipes
+Remove erroneous `SHALLOW_WATER` flag from `t_swater_dp`
+refactor item::*_resist, fix bugs related to clothing mods and outfit bodypart coverage
+Add missing flags to Predator Zombie evolution line
+Limit glare max duration
+Add missing space inside cashcard displayname
+stored calories cap instead of integer overflowing
+Fixes a bunch of disassembly recipes
+Adding recipe for craft freezer and adding ability to disassemble it, reducing taking dawn time
+fix for encumbrance of pals attached to riot armor suit
+Fix monster bleeding on all damage types
+fix id conflict with ap_compact_asrg_containment
+fixes the wretch’s raptor spawning ability.
+Adding plastic chunks to breaks_into of freezers and fridges and their mini variants. Editing player placeable appliance variants
+Make both Hub 01 modular defense ponchos' holster pocket compatible with all HWP conversion kits.
+Disassembly doesn't return items with UNRECOVERABLE flag
+Fixes numbers for normal bread and unifies charges across others
+Fixed inability to move down when standing on stairs next to a ledge or when flying a helicopter
+#64038
+Dissecting mutant mammals now yields samples
+Make 'Weak' conflict with other Strength-affecting mutations
+#64138
+Correct description of 'Hooves' mutation
+Fixed autoforage doesn't work on furniture with `harvest_furn_nectar` examine function
+Proofreading the descriptions of martial arts and their techniques to eliminate grammar & punctuation errors
+Spawn `liq_bandage_spray`, not `liq_bandage`
+Don't use `cruise_thrust()` on towed vehicles
+Update `cac2powder` recipe due to coal stat changes
```

Comment:
```suggestion
Crafting GUI: show how much recipe makes for non-charge items
Monsters add weight to vehicle when on boardable parts
Make AUTO_PICKUP_SAFEMODE also consider ignored mobs
Reset daily health at the end of each day
Food inside sealed containers is properly labeled as such in the [E]ating menu
Prevent autodrive from dropping vehicles in holes
Allow room for starting NPC when picking player starting position
Determine how much you can squish a soft container by its contents
Avoid updating weather every turn, respect weather-specific duration settings
No cooldown in sfx::do_obstacle(); furniture-specific obstacle sounds not used
mutate_towards accounts for bionics which CANCEL but don't CONFLICT with mutations
Fix files after a symlink in a directory all treated as symlink on Windows
Fixes marina spawns
Fix NPCs unable to trade items away if they have no pockets
Display IME candidate list and composition text correctly on Windows
Fix dark gray in the ncurses client for terminal emulators that support 256 colors.
Fixes mobile home park road connections
Flush map buffers after failing to create starting location
Smart controller one engine support
refactor item::*_resist, fix bugs related to clothing mods and outfit bodypart coverage
Stored calories cap instead of integer overflowing
Fix monster bleeding on all damage types
Disassembly doesn't return items with UNRECOVERABLE flag
Dissecting mutant mammals now yields samples
```

Reply:
You can remove this "Dissecting mutant mammals now yields samples" they already did this is just a small sample of missed/new mutants missing samples.